<launch>
	<rosparam command="load" file="$(find spacejockey)/config/planner.yaml" ns="planner" />
	<rosparam command="load" file="$(find spacejockey)/config/joints.yaml" />
	<param name="robot_description" textfile="$(find spacejockey)/resources/spacejockey.urdf" />
	<node name="waypoint_planner" pkg="spacejockey" type="waypoint_planner.py" required="true" />
	<!-- <node name="planner_broadcaster" pkg="spacejockey" type="planner_broadcaster.py"/> -->
	<node name="minor_planner" pkg="spacejockey" type="minor_planner.py"/>
	<node pkg="joint_state_publisher" type="joint_state_publisher" name="rob_joint_pub">
		<rosparam command="load" file="$(find spacejockey)/config/jsp_options.yaml" />
	</node>
	<node pkg="spacejockey" type="control_bridge.py" name="ctl_bridge" />
	<node name="serial_node" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM0" />

	<node pkg="robot_state_publisher" type="robot_state_publisher" name="rob_st_pub" />
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find spacejockey)/resources/spacejockey.rviz" required="true" />
</launch>
